experiments/standalone/Vegetation/varying_lai.jl

```diff
@@ -73,41 +77,26 @@ canopy = ClimaLand.Canopy.CanopyModel{FT}(
     hydraulics,
 );
 
-Y, p, coords = ClimaLand.initialize(canopy)
-exp_tendency! = make_exp_tendency(canopy);
-imp_tendency! = make_imp_tendency(canopy)
-jacobian! = make_jacobian(canopy);
-jac_kwargs =
-    (; jac_prototype = ClimaLand.FieldMatrixWithSolver(Y), Wfact = jacobian!);
-
 
 (; retention_model, ν, S_s) = canopy.hydraulics.parameters;
 ψ_leaf_0 = FT(-2e5 / 9800)
 S_l_ini = inverse_water_retention_curve(retention_model, ψ_leaf_0, ν, S_s)
-Y.canopy.hydraulics.ϑ_l.:1 .= augmented_liquid_fraction.(ν, S_l_ini)
 
-t0 = 0.0
-N_days = 364
-tf = t0 + 3600 * 24 * N_days
-dt = 225.0;
-evaluate!(Y.canopy.energy.T, atmos.T, t0)
-set_initial_cache! = make_set_initial_cache(canopy)
-set_initial_cache!(p, Y, t0);
+function set_ic!(Y, p, t0, model)
+    Y.canopy.hydraulics.ϑ_l.:1 .= augmented_liquid_fraction.(ν, S_l_ini)
+    evaluate!(Y.canopy.energy.T, atmos.T, t0)
+end
```

Comment:
I think it's cleaner and less error-prone to avoid a closure and move parameter access inside of `set_ic!`. And then you can delete the access of these parameters outside the function (lines 81-83)
```suggestion
function set_ic!(Y, p, t0, model)
    (; retention_model, ν, S_s) = model.hydraulics.parameters
    ψ_leaf_0 = FT(-2e5 / 9800)
    S_l_ini = inverse_water_retention_curve(retention_model, ψ_leaf_0, ν, S_s)
    
    Y.canopy.hydraulics.ϑ_l.:1 .= augmented_liquid_fraction.(ν, S_l_ini)
    evaluate!(Y.canopy.energy.T, atmos.T, t0)
end
```

Reply:
Actually this applies to pretty much every experiment, so why don't we leave it as an improvement for a followup PR. Could you open an issue for it please?